### ISSUE TITLE
Hide List Stages in specific View

### ISSUE THREAD

**Issue by Us3r-gitHub** [tillywork/tillywork#86]
I think different views can be display different list stages

Is blocked by #50

**Comment by codonade** [comment]
I think this kind of relates to #52, since `Done` is a list stage.

**Comment by codonade** [comment]
I think this kind of relates to #52 since `Done` is a list stage, so we can just hide it if this feature gets implemented.

**Comment by Us3r-gitHub** [comment]
I mean, we can hide in specific view

Let's say, we can hide `Done` in board view but keep it displayed in table view

**Comment by codonade** [comment]
Yeah I understand what you're trying to do. Just saying that implementing this would also resolve another issue.

**Comment by MoShizzle** [comment]
Reopening as having the ability to hide/unhide list stages in specific views can be useful

**Comment by MoShizzle** [comment]
> I think this kind of relates to #52 since `Done` is a list stage, so we can just hide that stage if this feature gets implemented.

I think this is a separate issue, since hiding specific list stages in certain views vs just hiding completed stages are two different use cases

**Comment by MoShizzle** [comment]
This was solved in #111, you can now filter on list stages, so you can do this:
<img width="761" alt="Screenshot 2024-07-03 at 7 39 26 PM" src="https://github.com/tillywork/tillywork/assets/28713654/896d1efa-6710-48c3-8e79-ee013206af52">

**Comment by Us3r-gitHub** [comment]
Can we reopen this?

![image](https://github.com/user-attachments/assets/39859463-8da1-4f23-9451-71dbdec9f5b5)
The group stage still exists.
Due to limitations in the filtering capability, it only hides cards and not the group stage
